src/single-post.ejs

```diff
@@ -0,0 +1,610 @@
+                            </div>
+                            <div class="post-facilites__facility">
+                                <i class="post-facilites__icon"><img
+                                        src="<%=require( "./icons/svgs/car-o.svg"  ).default%>" alt=""></i>
+                                <p class="post-facilites__label">پارکینگ</p>
+                            </div>
+                            <div class="post-facilites__facility post-facilites__facility--no-border ">
+                                <i class="post-facilites__icon"><img
+                                        src="<%=require( "./icons/svgs/cabinet-o.svg"  ).default%>" alt=""></i>
+                                <p class="post-facilites__label">انباری</p>
+                            </div>
+                        </div>
+                        <div class="post_description">
+                            <p class="post_description__title">
+                                توضیحات
+                            </p>
+                            <p class="post_description__detail">
+                                ⁦⚜نویدی دیگر از حاج رحیم قربانی ⁦⚜
+
+                                ⭐ پروژه‌ی تکرار نشدنی نشاط⭐
+
+                                ⭕امکانات رفاهی مجموعه⭕
+
+                                ✅- لابی لاکچری و بی نظیر با مساحت ۱۰۰۰ متر
+
+                                ✅- لابی من و لابی وُمن ۲۴ ساعته طبقه پنجم
+
+                                ✅- تیم نگهبانی و حراست
+
+                                ✅- 7 لاین آسانسور
+
+                                ✅- تالار تشریفاتی برای برگزاری مراسمات و میهمانی
+
+                                ✅- باشگاه بدن سازی آقایان و بانوان
+
+                                ✅- سالن بازی کودکان
+
+                                ✅- اتاق مانیتورینگ
+
+                                ✅ -کلوپ بیلیارد
+
+                                دسترسی آسان به مراکز خرید و بزرگراهای(سیمون بولیوار،اشرفی اصفهانی ،نیایش )
+
+                                ⭐⭐«اعتبار ما سال‌ها تجربه در امور ساخت و ساز⭐⭐
+                                ⭐⭐و خرید و فروش ملک در پایتخت است»⭐⭐
+
+                                ⁦✳️⁩به دلیل نشست مستقیم با سازنده⁦✳️⁩
+                                قیمت و شرایت پرداخت توافقی و قابل مذاکره
+
+                                ⁦✳️⁩دفتر فروش مستقیم سازنده⁦✳️⁩
+
+                                مدیر فروش پروژه‌ی «نشاط»
+
+                                09127357249☎️
+                                09127357249☎️
+                            </p>
+                        </div>
+                    </div>
+                    <div class="single-post__side single-post__side--left">
+                        <section class="carousel" aria-label="carousel" Tabindex="0">
+
+                            <a class="carousel__skip-link" href="#skip-link">Skip the Carousel</a>
+
+                            <div class="slides">
+                                <div class="slides-item slide-1" id="slide-1" tabindex="0">
```

Comment:
Same goes for these.  
Also the slide-1 classname is not acceptable at all. And as far as I can see, you haven't even used it